src/app/routes/getInitialData/index.js

```diff
@@ -18,7 +20,47 @@ const baseUrl = onClient()
   ? getBaseUrl(window.location.origin)
   : process.env.SIMORGH_BASE_URL;
 
-const getUrl = pathname => `${baseUrl}${pathname.replace(ampRegex, '')}.json`;
+export const validateRendererEnvironment = environment => {
+  const validRendererEnvironments = ['test', 'live'];
+
+  if (
+    !(
+      environment &&
+      validRendererEnvironments.includes(environment.toLowerCase())
+    )
```

Comment:
this is hard for me to parse. it helps when things like this are in a well name variable lke:

```js
if (isValidEnv) {
```

I also wouldn't bother correcting any user error casing issues with `environment.toLowerCase()`. I think if we can't type in 'test' or 'live' properly then it's fine not to do anything.

Reply:
I have removed this functionality, but good shout - something to bear in mind for future - ensuring the code is readable & maintainable in the future.